src/enkf/params.f90

```diff
@@ -254,6 +254,11 @@ module params
 ! write ensemble mean analysis (or analysis increment)
 logical,public :: write_ensmean = .false.
 
+! taper analysis ens perturbations at top of model (gfs only)
+logical, public :: taperanalperts = .false.
+real, public :: taperanalperts_akbot = 500.
```

Comment:
Should specify the intended precision:  `r_single`, `r_kind`, or ...?

Reply:
done by making r_kind